fixedwidth/src/test/java/org/apache/metamodel/fixedwidth/FixedWidthConfigurationTest.java

```diff
@@ -27,18 +27,18 @@
 	public void testToString() throws Exception {
 		assertEquals(
 				"FixedWidthConfiguration[encoding=UTF8, fixedValueWidth=10, valueWidths=[], columnNameLineNumber=1, failOnInconsistentLineWidth=true]",
-				new FixedWidthConfiguration(1, "UTF8", 10, true).toString());
+				new FixedWidthConfiguration(1, "UTF8", 10, true, false, true).toString());
 	}
 
 	public void testEquals() throws Exception {
 		FixedWidthConfiguration conf1 = new FixedWidthConfiguration(1, "UTF8",
-				10, true);
+				10, true, false, true);
```

Comment:
Please make sure there are overloaded constructors available for the old constructor signature. Then you also wont have to change this existing test.